src/main.py

```diff
@@ -83,4 +83,4 @@ def build(self):
 if __name__ == '__main__':
     if hasattr(sys, '_MEIPASS'):
         resource_add_path(os.path.join(sys._MEIPASS))
-    SpeechJokey().run()
\ No newline at end of file
+    SpeechJokey(kv_file="SpeechJokey.kv").run()
```

Comment:
That should not be necessary as Kivy associates the class name with the widget name automatically, but I accept it, since I dislike this magic binding behavior anyway.